jib-cli/src/test/java/com/google/cloud/tools/jib/cli/buildfile/FilePropertiesSpecTest.java

```diff
@@ -146,8 +146,11 @@ public void testFilePropertiesSpec_noEmptyValues() {
     public void testFilePropertiesSpec_nullOkay() throws JsonProcessingException {
       String data = fieldName + ": null";
 
-      mapper.readValue(data, FilePropertiesSpec.class);
-      // pass
+      FilePropertiesSpec parsed = mapper.readValue(data, FilePropertiesSpec.class);
```

Comment:
I’m more familiar with these types of  questions on values rather than the “Test.None.class” approach.